studio/pages/project/[ref]/auth/policies.tsx

```diff
@@ -16,24 +16,39 @@ const AuthPoliciesLayout = ({ children }: PropsWithChildren<{}>) => {
   return <div className="p-4">{children}</div>
 }
 
+/**
+ * Filter tables by table name and policy name
+ *
+ * @param tables list of PostgresTable
+ * @param policies list of policies
+ * @param keywords filter keywords
+ *
+ * @returns list of PostgresTable
+ */
 const onFilterTables = (
   tables: {
     name: string
-    policies: {
-      name: string
-    }[]
+  }[],
+  policies: {
+    name: string
+    table: string
   }[],
   keywords?: string
```

Comment:
just naming - its a little confusing

could we name this to `searchString` - `keywords` sounds like an array of strings

then rename `searchString` to `findSearchString` since its a function